source/backend/dal/Repositories/AcquisitionFileRepository.cs

```diff
@@ -143,7 +143,9 @@ public PimsAcquisitionFile Update(PimsAcquisitionFile acquisitionFile)
             }
 
             this.Context.Entry(existingAcqFile).CurrentValues.SetValues(acquisitionFile);
-            this.Context.UpdateChild<PimsAcquisitionFile, long, PimsAcquisitionFilePerson>(p => p.PimsAcquisitionFilePeople, acquisitionFile.Internal_Id, acquisitionFile.PimsAcquisitionFilePeople.ToArray());
+            this.Context.UpdateChild<PimsAcquisitionFile, long, PimsAcquisitionFilePerson>(p => p.PimsAcquisitionFilePeople, acquisitionFile.Internal_Id, acquisitionFile.PimsAcquisitionFilePeople.ToArray(), true);
+            this.Context.UpdateChild<PimsAcquisitionFile, long, PimsAcquisitionOwner>(p => p.PimsAcquisitionOwners, acquisitionFile.Internal_Id, acquisitionFile.PimsAcquisitionOwners.ToArray(), true);
+            this.Context.UpdateGrandchild<PimsAcquisitionFile, long, PimsAcquisitionOwner>(o => o.PimsAcquisitionOwners, oa => oa.Address, acquisitionFile.Internal_Id, acquisitionFile.PimsAcquisitionOwners.ToArray());
```

Comment:
Actually I don't think this is correct to chain update child and updategrandchild like this, I think you should be using one or the other here.

Reply:
Yeah, I checked this out and tried it, throws an exception when adding a net new owner during an update.

Reply:
Updated